articles/azure-monitor/app/opencensus-python-request.md

```diff
@@ -113,6 +113,48 @@ First, instrument your Python application with latest [OpenCensus Python SDK](./
     config = Configurator(settings=settings)
     ```
 
+## Tracking FastAPI applications
+
+1. The following dependencies are required: 
+    - fastapi
+    - uvicorn
+
+2. Add FastAPI middleware.
+
+3. Set span kind server, `span.span_kind = SpanKind.SERVER`.
+
+```python 
+app = FastAPI()
+logger = logging.getLogger(__name__)
+
+HTTP_URL = COMMON_ATTRIBUTES['HTTP_URL']
+HTTP_STATUS_CODE = COMMON_ATTRIBUTES['HTTP_STATUS_CODE']
+
```

Comment:
Add a comment indicating that the below code is the `middleware`.

Reply:
Thank you, updated. let me know if further change is required.